### ISSUE TITLE
[Effect Suggestion] Become a modder

### ISSUE THREAD

**Issue by Aran8276** [gta-chaos-mod/ChaosModV#1101]
**Name of effect**
Modder POV
Modder plz drop money
Is that a mod menu?

**Duration if timed effect (long or short timed?)**
 Long

**Effect Description**
Gave player a mod limited mod menu, each mod effect cost $50,000

**Comment by BaronVonDelta** [comment]
Might be good to have mod effects not cost anything since effects like poor boy can mean you can't do anything with it. Also might be a fun touch for the effect to end with a "You have been banned from GTA Online" screen or something to that effect.

**Comment by LordRygar** [comment]
I agree with the mod menu, don't charge, and the banned from online parts.
